test/navigation/NavigationSpec.scala

```diff
@@ -519,7 +519,11 @@ class NavigationSpec extends SpecBase {
       }
 
```

Comment:
We also need a test for when the appeal is >30 days late going to making a late appeal page rather than CYA page

Reply:
Added a test for the same.